server/routes.go

```diff
@@ -602,6 +602,7 @@ func streamResponse(c *gin.Context, ch chan any) {
 			return false
 		}
 
+		// Delineate chunks with new-line delimiter
```

Comment:
We can omit this comment since it's in the `api.md` docs
```suggestion
```

Reply:
I think it will help keep this behavior whenever someone is refactoring the code. Ideally, I would have implemented a test that checks for this and so it would result in an automated failure -- however that requires more effort.

So we should keep it. That being said, I am happy to defer to your preference if you feel strongly!

Reply:
Sounds great